ref.yml

```diff
@@ -0,0 +1,232 @@
+keys:
+- fullname:
+    required: true
+    used:
+    - README.md
+    - index.md
+- shortname:
+    required: true
+    used:
+    - README.md
+    - index.md
+    - coq.opam
+    - extracted.opam
+    - .travis.yml
+    - coq-action.yml
+    - config.yml
+    - default.nix
+- organization:
+    required: true
+    used:
+    - README.md
+    - index.md
+    - coq.opam
+    - extracted.opam
+- travis:
+    type: bool
+    used:
+    - README.md
+- action:
+    type: bool
+    used:
+    - README.md
+- circleci:
+    type: bool
+    used:
+    - README.md
+- community:
+    type: bool
+    used:
+    - README.md
+    - index.md
+- coqdoc:
```

Comment:
I hadn't realized we had this boolean variable available already. We should generate `index.md` only if it is set to `true`, right?